composer.json

```diff
@@ -21,17 +21,17 @@
     ],
     "require": {
         "php": "^7.4 || ^8.0",
-        "silverstripe/admin": "^1.6",
-        "silverstripe/framework": "^4.10",
-        "silverstripe/versioned": "^1.6",
-        "silverstripe/graphql": "^4",
-        "silverstripe/vendor-plugin": "^1",
+        "silverstripe/admin": "^1.6 | ^2",
+        "silverstripe/framework": "^4.10 | ^5",
+        "silverstripe/versioned": "^1.6 | ^2",
+        "silverstripe/graphql": "^4 | ^5",
+        "silverstripe/vendor-plugin": "^1 |^2",
```

Comment:
Oh! Yeah this dual support might not work with `@silverstripe/webpack-config` since CMS 4 ships with different webpack externals than CMS 5